RadarSDK/RadarUtils.m

```diff
@@ -156,6 +158,57 @@ + (NSString *)dictionaryToJson:(NSDictionary *)dict {
     } 
 }
 
++ (BOOL)isJailbroken {
+    NSArray *jailbreakFilePaths = @[
+        @"/Applications/Cydia.app",
+        @"/Library/MobileSubstrate/MobileSubstrate.dylib",
+        @"/bin/bash",
+        @"/usr/sbin/sshd",
+        @"/etc/apt",
+        @"/private/var/lib/apt/"
+    ];
+
+    for (NSString *path in jailbreakFilePaths) {
+        if ([[NSFileManager defaultManager] fileExistsAtPath:path]) {
+            NSLog(@"failed check 1");
+            return YES;
+        }
+    }
+    
+    NSString *testPath = @"/private/JailbreakTest.txt";
+    NSData *data = [@"This is a test." dataUsingEncoding:NSUTF8StringEncoding];
+    
+    NSError *error;
+    [data writeToFile:testPath options:NSDataWritingAtomic error:&error];
+    
+    // If no error occurred, the device is likely jailbroken
+    if (!error) {
+        NSLog(@"failed check 2");
+        [[NSFileManager defaultManager] removeItemAtPath:testPath error:nil];
+        return YES;
+    }
```

Comment:
The first test makes sense to me.

For this test here, I'd love to read up on why this test works — is there a SO or other resource you can point me to?

Reply:
This doesn't actually work, reverting